libs/domain/picking/offline/data-plugin.ts

```diff
@@ -28,14 +31,30 @@ export class OfflineDataPlugin extends ExecPlugin {
       const env = injector.inject(AppEnvironment, {} as AppEnvironment);
 
       const authService = injector.inject(OauthService);
+      const routerService = injector.inject(RouterService);
+      const oauthConfig = injector.inject(OauthServiceConfig);
 
       this.subscription = authService
         .isAuthenticated()
         .pipe(
+          distinctUntilChanged(),
```

Comment:
I don't think this will prevent multiple DB cleanups/populations as if you just reload the page it will still perform full DB drop and repopulate the data.
As we discussed before this data fetching logic should be performed _only_ after the login and not generically on the application startup.